omod/src/test/java/org/openmrs/module/idgen/web/controller/IdentifierSourceRestControllerTest.java

```diff
@@ -101,7 +101,11 @@ public void shouldUploadReservedIdentifiers() throws Exception {
         MockHttpServletRequest getRequestAfterUpload = newGetRequest(getURI() + "/" + SEQUENTIAL_IDENTIFIER_SOURCE_UUID);
         getRequestAfterUpload.addParameter("v", "custom:(reservedIdentifiers)");
         SimpleObject identifiersAfterUpload = deserialize(handle(getRequestAfterUpload));
-        assertEquals("[1, 2, 3, 4]", identifiersAfterUpload.get("reservedIdentifiers").toString());
```

Comment:
Was the failure caused by the identifiers being 1,2,3,4 instead of 100,200,300,400?

Reply:
@dkayiwa No it was the `assertEqual` which was causing the issue. I replaced it with `assertThat` which checks that the result of the request contains the identifiers expected. I changed the identifiers to test for different values. 

<img width="1104" alt="test error" src="https://user-images.githubusercontent.com/25571987/32827377-074ae23e-c9fd-11e7-8a8e-cf124e30c72d.png">

Reply:
Then why did you change the values?

Reply:
@dkayiwa I first tested the original values and they were working. I then put a wrong set of values and the test failed as expected. I decided to use a different set of values just to confirm.

Reply:
The change in this pull request should only be that which fixes the reported problem. If a change of values is not part of the fix, then revert to the old values. We want to make it absolutely clear to whoever looks at the commit to tell what exactly the fix was.

Reply:
@dkayiwa changed